ledger/accountdb_test.go

```diff
@@ -369,6 +372,53 @@ func checkAccounts(t *testing.T, tx *sql.Tx, rnd basics.Round, accts map[basics.
 	d, err := aq.lookup(randomAddress())
 	require.NoError(t, err)
 	require.Equal(t, d, basics.AccountData{})
+
+	onlineAccounts := make(map[basics.Address]*onlineAccount)
+	for addr, data := range accts {
+		if data.Status == basics.Online {
+			onlineAccounts[addr] = accountDataToOnline(&data)
+		}
+	}
+
+	proto := config.Consensus[protocol.ConsensusCurrentVersion]
+	for i := 0; i < len(onlineAccounts); i++ {
+		dbtop, err := accountsOnlineTop(tx, 0, uint64(i))
+		require.NoError(t, err)
+		require.Equal(t, len(dbtop), i)
```

Comment:
nit: swap arguments of require.Equal : 
```
require.Equal(t, i, len(dbtop))
```
in case of an error, it would printout that the second argument is the "expected" and the third is the "actual"